pretyping/evarconv.ml

```diff
@@ -526,31 +533,35 @@ and evar_eqappr_x ?(rhs_is_already_stuck = false) flags env evd pbty
   let quick_fail i = (* not costly, loses info *)
     UnifFailure (i, NotSameHead)
   in
-  let miller_pfenning on_left fallback ev lF tM evd =
+  let miller_pfenning l2r fallback ev lF tM evd =
```

Comment:
I'm not so sure l2r is an improvement. If I understand it well, here we have a boolean to tell us if the problem is `?e x = t` or `t = ?e x`. I guess `on_left` meant that the evar was on the left. If so, I think `evar_on_the_left` would be better.

Reply:
In other places, `l2r` mean "remember to swap". In any case I don't get the intuition in the direction.

Reply:
It actually means "remember to swap", otherwise, we would not have to remember that the evar was originally not on the left.